extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -42,18 +39,16 @@ export class AmpScrollableCarousel extends BaseCarousel {
     /** @private {number} */
     this.pos_ = 0;
 
+    this.oldPos_ = 0;
+
     /** @private {!Array<!Element>} */
     this.cells_ = this.getRealChildren();
 
     /** @private {!Element} */
     this.container_ = this.element.ownerDocument.createElement('div');
     st.setStyles(this.container_, {
-      whiteSpace: 'nowrap',
-      position: 'absolute',
-      zIndex: 1,
-      top: 0,
-      left: 0,
-      bottom: 0,
+      'whiteSpace': 'nowrap',
+      'overflow-x': 'auto',
```

Comment:
please also add `'overflow-y': 'hidden'`. It's important because when you set `overflow-x: auto`, the `overflow-y` is implicitly set to auto as well.

Also, please add `-webkit-overflow-scrolling: touch` as well for it to work on Safari.